packages/block-library/src/social-link/block.json

```diff
@@ -34,6 +34,27 @@
 		"html": false,
 		"interactivity": {
 			"clientNavigation": true
+		},
+		"spacing": {
+			"margin": true,
+			"padding": true,
+			"units": [ "px", "em", "rem", "vh", "vw" ],
```

Comment:
Do we need custom definitions of available spacing units here?

Reply:
I guess not.